### PR TITLE
Registery changes

### PR DESCRIPTION
**Pull request by siba-tunerlabs**

### CODE REVIEW

File: src/components/entity-listing/entity-listing.ts [hunk 1]
```diff
@@ -102,13 +102,22 @@ export class EntityListingComponent {
     })
   }
 
-  openMenu(event, programIndex, assessmentIndex, entityIndex) {
-    console.log("emitting called open menu")
+  openMenu(event, programIndex, assessmentIndex, entityIndex , submissionId) {
```
Comment (kiranharidas187) [inline src/components/entity-listing/entity-listing.ts]: use es6

File: src/components/menu-item/menu-item.ts [hunk 2]
```diff
@@ -94,7 +105,12 @@ export class MenuItemComponent {
     this.appCtrl.getRootNav().push('ParentsListPage', {
```
Comment (kiranharidas187) [inline src/components/menu-item/menu-item.ts]: rename to registry list

File: src/pages/home/home.ts [hunk 3]
```diff
@@ -85,13 +85,14 @@ export class HomePage {
     // this.subscription = this.events.subscribe('localDataUpdated', () => {
     //   this.getLocalSchoolDetails();
     // });
+    this.userData = this.currentUser.getCurrentUserData();
```
Comment (kiranharidas187) [inline src/pages/home/home.ts]: remove

File: src/pages/individual-listing/individual-listing.ts [hunk 4]
```diff
@@ -125,30 +127,45 @@ export class IndividualListingPage {
...
     var schoolIndex = event.entityIndex;
-    console.log(JSON.stringify(this.programs));
+    var submissionId = event.submissionId;
+    let showMenuArray;
+      console.log(submissionId + "submissionid");
+    this.localStorage.getLocalStorage(this.utils.getAssessmentLocalStorageKey(submissionId)).then(successData => { 
+      showMenuArray= successData.solution.registry ;
+      console.log("success data")
+      console.log(JSON.stringify(successData.solution.registry ) + "         REGISTERY ARRAY     " + submissionId)
+      let popover = this.popoverCtrl.create(MenuItemComponent, {
```
Comment (kiranharidas187) [inline src/pages/individual-listing/individual-listing.ts]: remove from individual

File: src/pages/registry-form/registry-form.ts [hunk 5]
```diff
@@ -21,15 +21,29 @@ export class RegistryFormPage {
...
 
   constructor(public navCtrl: NavController,
     public navParams: NavParams,
     private viewCntrl: ViewController, private ngps: NetworkGpsProvider,
     private events: Events, private localStorage: LocalStorageProvider,
     private utils: UtilsProvider, private apiService: ApiProvider) {
     this.schoolId = this.navParams.get('_id');
+    this.submissionId = this.navParams.get('submissionId');
     this.schoolName = this.navParams.get('name');
+    console.log(this.navParams.get('name'));
```
Comment (kiranharidas187) [inline src/pages/registry-form/registry-form.ts]: remove logs

File: src/pages/registry-list/registry-list.ts [hunk 6]
```diff
@@ -47,28 +51,38 @@ export class ParentsListPage {
...
+      if (entityDetails) {
+        this.entityDetails = entityDetails;
         this.showUploadBtn = this.checkForUploadBtn();
       }
     }).catch(error => {
 
     })
 
-    this.storage.get(this.registryType + 'Details_' + this.schoolId).then(registryData => {
+    this.storage.get(this.registryType + 'Details_' + this.entityId).then(registryData => {
```
Comment (kiranharidas187) [inline src/pages/registry-list/registry-list.ts]: use submissionId

File: src/providers/assessment-service/assessment-service.ts [hunk 7]
```diff
@@ -149,7 +152,42 @@ export class AssessmentServiceProvider {
...
+  var myEvent = event.event;
+  var  programIndex =  event.programIndex;
+  var assessmentIndex = event.assessmentIndex ;
+  var schoolIndex = event.entityIndex;
+  var submissionId = event.submissionId;
+  let showMenuArray;
+  var solutionId = event.solutionId;
+  var parentEntityId = event.parentEntityId;
+  var createdByProgramId = event.createdByProgramId;
+  console.log(" id related to this page");
```
Comment (kiranharidas187) [inline src/providers/assessment-service/assessment-service.ts]: remove logs

File: src/providers/assessment-service/assessment-service.ts [hunk 8]
```diff
@@ -149,7 +152,42 @@ export class AssessmentServiceProvider {
 
   });
 }
+openMenu(event , programs) {
+  var myEvent = event.event;
```
Comment (kiranharidas187) [inline src/providers/assessment-service/assessment-service.ts]: use let / const instead of var

File: src/components/menu-item/menu-item.ts (hunk 2 above)
Comment (siba-tunerlabs) [reply src/components/menu-item/menu-item.ts]: done

File: src/pages/home/home.ts (hunk 3 above)
Comment (siba-tunerlabs) [reply src/pages/home/home.ts]: done

File: src/pages/individual-listing/individual-listing.ts (hunk 4 above)
Comment (siba-tunerlabs) [reply src/pages/individual-listing/individual-listing.ts]: done

File: src/pages/registry-list/registry-list.ts (hunk 6 above)
Comment (siba-tunerlabs) [reply src/pages/registry-list/registry-list.ts]: done

File: src/providers/assessment-service/assessment-service.ts (hunk 7 above)
Comment (siba-tunerlabs) [reply src/providers/assessment-service/assessment-service.ts]: done

File: src/providers/assessment-service/assessment-service.ts (hunk 8 above)
Comment (siba-tunerlabs) [reply src/providers/assessment-service/assessment-service.ts]: ok
done

**Pull request merged**